noxfile.py

```diff
@@ -142,10 +142,26 @@ def test(session, pandas_version: str):
 @nox.session(reuse_venv=True)
 def docs(session):
     # Run this so users get an error if they don't have Pandoc installed.
-
     session.install("-r", "docs/requirements-docs.txt")
     session.install(".")
 
     session.cd("docs")
     session.run("make", "clean", external=True)
     session.run("make", "html", external=True)
+
+
+@nox.session
```

Comment:
Let's add comment here for developers not to get confused.

Reply:
Ok!